.github/in-cluster-test-scripts/aks-byocni-install.sh

```diff
@@ -10,4 +10,5 @@ cilium install \
   --wait=false \
   --set loadBalancer.l7.backend=envoy \
   --set tls.secretsBackend=k8s \
-  --set bpf.monitorAggregation=none
+  --set bpf.monitorAggregation=none \
+  --set ipam.operator.clusterPoolIPv4PodCIDRList=192.168.0.0/16" # To avoid clashing with the default Service CIDR of AKS (10.0.0.0/16)
```

Comment:
This broke CI on AKS due to the trailing `"`. Unfortunately this wasn't caught in CI because we would have needed to follow the workflow testing instructions here: https://github.com/cilium/cilium-cli/blob/918ae165b6995780e3f57aa003d6c888ccf3cecd/.github/workflows/aks-byocni.yaml#L5-L17

Fixed in https://github.com/cilium/cilium-cli/pull/2452